pkg/utils/types.go

```diff
@@ -271,3 +272,79 @@ func PriorityClass(priorityclassName string, priorityValue int32, globalDefault
 		Description:   description,
 	}
 }
+
+func PolicyRule(apiGroups, resources, resourceNames, verbs []string) rbac.PolicyRule {
```

Comment:
Please use accepted golang  style conventions to name these functions  `NewPolicyRule`, `NewPolicyRules`, `NewSubject`, etc